### DISCUSSION TITLE
localAI  maxtoken

### DISCUSSION THREAD

**Discussion by sunbopython** [langgenius/dify#3739] [General]
![image](https://github.com/langgenius/dify/assets/21340635/4d09403c-82f1-4dff-8db3-95c6033a24bc)
配置的local AI 模型 这个地方中的最大生成长度如何调整

**Comment by YidaHu** [comment]
@sunbopython 
<img width="1076" alt="image" src="https://github.com/langgenius/dify/assets/13404367/b036a11e-91e5-47fc-918c-b1361c168fda">
